apps/shipments/views/shipment.py

```diff
@@ -178,6 +179,38 @@ def tracking(self, request, version, pk):
 
         raise NotFound("No tracking data found for Shipment.")
 
+    @method_decorator(cache_page(60 * 60, remember_all_urls=True))  # Cache responses for 1 hour
```

Comment:
You have this view cached for an hour but you never invalidate the cache which means this view will always be stale, potentially up to an hour stale. Please see the cache invalidation logic in trackingdata_post_save